src/domain/hanke/hankeUsers/hankeUsersApi.ts

```diff
@@ -1,5 +1,5 @@
 import api from '../../api/api';
-import { HankeUser, IdentificationResponse, SignedInUser } from './hankeUser';
+import { HankeUser, IdentificationResponse, SignedInUser, SignedInUserByHanke } from './hankeUser';
 
 export async function getHankeUsers(hankeTunnus: string) {
   const { data } = await api.get<{ kayttajat: HankeUser[] }>(`hankkeet/${hankeTunnus}/kayttajat`);
```

Comment:
Should `<{ kayttajat: HankeUser[] }>` be a type like `<SignedInUserByHanke>` and `<IdentificationResponse>` are?

Reply:
You mean something like `type HankeUsers = HankeUser[]` ?

I guess it could be used for consistency in this file. But maybe I don't want to change that in this pr as it would lead to changes that are somewhat unrelated to this task (if I understood correctly).